### ISSUE TITLE
Install torchvision 0.8.1

### ISSUE THREAD

**Issue by yaoshanliang** [ultralytics/yolov5#3793]
## ❔Question
When I run `pip install -r requirements.txt`, it has installed torch 1.7.0 and torchvision 0.10.0. 
However, torchvision 0.10.0 does not work with torch 1.7.0. The I tried `pip install torchvision==0.8.1`, it shows that `Could not find a version that satisfies the requirement torchvision ==0.8.1 (from versions 0.1.6...0.9.1, 0.10.0)`. 0.8.1 is not shown it that list.

**Comment by glenn-jocher** [comment]
@yaoshanliang see https://pytorch.org/get-started/locally/ for PyTorch installation help

**Comment by gayathri81** [comment]
@yaoshanliang  can you please share fix to install torchvision==0.8.1

**Comment by Symbadian** [comment]
> ```shell
> pip install -r requirements.txt
> ```

Hi @gayathri81, have you gotten a work around for the challenge with torchvision

```
pip install -r requirements.txt
CUDA_VISIBLE_DEVICES="0,1,2" python3 train.py --epoch 2 --cache --multi-scale --data coco128.yaml --weights yolov5n.pt --img 640
```
I tried everything I know and somehow nothing is working and I'm sure this is the lack of my understanding...

Can you guide me, if you've got the solution, please? my errors are below!

```
requirements: YOLOv5 requirement "torchvision>=0.8.1" not found, attempting AutoUpdate...
ERROR: Could not find a version that satisfies the requirement torchvision>=0.8.1 (from versions: 0.1.6, 0.1.7, 0.1.8, 0.1.9, 0.2.0, 0.2.1, 0.2.2, 0.2.2.post2, 0.2.2.post3)
ERROR: No matching distribution found for torchvision>=0.8.1
requirements: ❌ Command 'pip install "torchvision>=0.8.1"  ' returned non-zero exit status 1.
YOLOv5 🚀 v7.0-71-gc442a2e Python-3.7.16 torch-1.12.1 CPU
```

The code runs in CPU nevertheless I am initiating the GPU Command!!! I am not sure how to turn this on, based on my google research... Everything I've learnt so far is through trial and error and well google!

Please assist, I'm really struggling here!
